community/github/issues.md

```diff
@@ -0,0 +1,82 @@
+# pyOpenSci GitHub issue process
+
+When possible, pull request and issue submissions and reviews should follow
+standard open source workflows. Below are guidelines for handling issues.
+
+## Guidelines for New Issues
+
+
+**Issues in should be as specific as possible:** specifics within an issue help both our future selves and also outside contributors understand the goal or
+desired outcome associated with addressing the issue. This is important both internally and for issues that we tag as `help-wanted`, which we hope community members will address in pyOpenSci sprints.
+
+To ensure an issue is well-written and specific, include the following details:
+
+- **Clear Title:** A concise and descriptive title summarizing the issue or feature request helps us scan through issues and understand what each issue is about. Below are some example titles that are specific
+
+    * `Bug: broken link in link-to-page-here / name of page / document etc`
+    * `Fix: confusing paragraph on Python packaging with Hatch`
+    * `Add: page on using pixi for dependencies`
+
+- **Description:** A detailed explanation of the issue or feature request, including context, background information, and the reason for the request. Explain why the issue is important and what problem it solves.
+
+- **Screenshots/Code Samples:** Screenshots, code snippets, links, or any other relevant files that can help in understanding the issue better.
+
+- **Possible Solutions/Recommendations:** Any ideas or suggestions for how to address the issue, which can help guide contributors towards a solution.
+
+- **Related Issues or Pull Requests:** References to any related issues or pull requests, providing additional context and understanding of the broader scope of the issue.
+
+### If you are reporting a code bug
+
+* **Steps to Reproduce (for bugs):** A step-by-step guide on how to reproduce the issue, including relevant code snippets, commands, or configurations.
+
+* **Expected vs. Actual Behavior (for bugs):** A description of what you expected to happen and what actually happened. This helps in understanding the discrepancy and the impact of the bug.
+
+* **Environment Details (for bugs):** Details about the environment where the issue was observed, such as operating system, Python version, library versions, and any other relevant software/hardware details.
```

Comment:
would it be helpful to tell people where to get some of this information? I'm thinking of beginners who might be using a tutorial, and not necessarily know how to get their Python or library versions, and/or know what additional software/hardware details might be helpful.

Reply:
normally you would do that in an issue template - where you'd have -- provide conda information for instance and then you have a code example. It will be different for each repo depending on what the repo is - ie sphinx book vs jekyll website ruby vs python. 

the ideal appoach would be to use issue forms for this.